Dockerfile

```diff
@@ -1,69 +1,96 @@
+# syntax=docker.io/docker/dockerfile:1.7-labs
 # Pelican Production Dockerfile
 
 # ================================
-# Stage 1: Build PHP Dependencies
+# Stage 1: Build PHP Base Image
 # ================================
-FROM --platform=$TARGETOS/$TARGETARCH php:8.3-fpm-alpine AS composer
+FROM --platform=$TARGETOS/$TARGETARCH php:8.3-fpm-alpine AS base
+
+ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
```

Comment:
No need for this. The base image comes with docker-php-ext-install.

Reply:
This is different, it's just a wrapper for the docker-php-ext-install script that additionally installs/removes build dependencies. I figured it's easier to make any changes to the extensions with no risk of forgetting to update dependencies.

Reply:
oké then this is fine you can mark this as resoloved

Reply:
Just for some additional info, I actually found that tool on the [PHP Docker page](https://hub.docker.com/_/php/) under `PHP Core Extensions`. It seemed handy, so I figured it was a good idea.